### PR TITLE
Feature/front and backend/version

### PR DESCRIPTION
**Pull request by urasakikeisuke**
## Pull Request Type

- [x] :bug: `fix` : (bug fix for the user, not a fix to a build script)
- [ ] :adhesive_bandage: `change` : (simple fix for a non-critical issue)
- [x] :sparkles: `feat` : (new feature for the user, not a new feature for build script)
- [ ] :memo: `docs` : (changes to the documentation)
- [ ] :recycle: `refactor` : (refactoring production code, eg. renaming a variable)
- [ ] :pencil2: `style` : (formatting, missing semi colons, etc; no production code change)
- [ ] :rotating_light: `test` : (adding missing tests, refactoring tests; no production code change)
- [ ] :package: `chore` : (updating grunt tasks etc; no production code change)
- [ ] :rewind: `reverts` : (canceling commits or pull requests)

## Description

Fixed some minor bugs and support version control.

## Pre-Review Checklist for the Pull Request Author

- [x] Code follows coding-guidelines
- [x] Assign Pull Request to reviewer

## Checklist for the Pull Request Reviewer

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows coding-guidelines
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] Pull Request title describes the changes

## Post-Review Checklist for the Pull Request Author

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write release notes

### CODE REVIEW

File: aicon/frontend/static/js/functions.js [hunk 1]
```diff
@@ -553,21 +553,23 @@ $('#communication_partner').focusout(function(){
...
-        data: send_data,
+        data: check_version_data,
         dataType: "json",
         timeout: 10000,
         async: true, // 同期通信  false:同期  true:非同期
         contentType: "application/json; charset=utf-8",
     })
-        .done(function () {
+        .done(function (r_data) {
+            console.log(r_data);
```
Comment (nakajima-john-shotaro) [inline aicon/frontend/static/js/functions.js]: 消してください

**Review by nakajima-john-shotaro — CHANGES_REQUESTED** [review]
Please one more.

**Review by urasakikeisuke — COMMENTED** [review]
AreYouForRealCatGIF

File: aicon/frontend/static/js/functions.js (hunk 1 above)
Comment (urasakikeisuke) [reply aicon/frontend/static/js/functions.js]: 申し訳ございませんでした。今後はこのようなことのないようにより一層精進していく所存でございます。

Comment (nakajima-john-shotaro) [reply aicon/frontend/static/js/functions.js]: Aazninteen07CatMemeFaceGIF

**Review by nakajima-john-shotaro — APPROVED** [review]

**Pull request merged**